<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountTab</class>
 <widget class="QWidget" name="AccountTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>1253</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblFilterText">
       <property name="text">
        <string>Search:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterText"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QTableView" name="accounts"/>
     </item>
     <item>
      <widget class="QWidget" name="details" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="lblDetailsName">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="detailsName">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblDetailsOwner">
          <property name="text">
           <string>Owner:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="detailsOwner">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblDetailsInitialBalance">
          <property name="text">
           <string>Initial Balance:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="detailsInitialBalance">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> â‚¬</string>
          </property>
          <property name="minimum">
           <double>-999999999.990000009536743</double>
          </property>
          <property name="maximum">
           <double>999999999.990000009536743</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblDetailsIban">
          <property name="text">
           <string>IBAN:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="detailsIban">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblDetailsBic">
          <property name="text">
           <string>BIC:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="detailsBic">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblDetailsAccountNumber">
          <property name="text">
           <string>Account Number:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="detailsAccountNumber">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblDetailsBankCode">
          <property name="text">
           <string>Bank Code:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="detailsBankCode">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblDetailsTags">
          <property name="text">
           <string>Tags:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="TagEdit" name="detailsTags">
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="moreAccountTags">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>17</width>
            <height>501</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="mergeAccounts">
          <property name="text">
           <string>Merge Accounts</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>src/ui/TagEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filterText</tabstop>
  <tabstop>detailsName</tabstop>
  <tabstop>detailsOwner</tabstop>
  <tabstop>detailsInitialBalance</tabstop>
  <tabstop>detailsIban</tabstop>
  <tabstop>detailsBic</tabstop>
  <tabstop>detailsAccountNumber</tabstop>
  <tabstop>detailsBankCode</tabstop>
  <tabstop>detailsTags</tabstop>
  <tabstop>mergeAccounts</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
